client/gameObjects/Skeleton.js

```diff
@@ -46,10 +46,12 @@ const MIN_TICK = 8;
 const MAX_TICK = 12;
 
 const HEALTH_BAR_OFFSET_X = -32;
-const HEALTH_BAR_OFFSET_Y = -56;
+const HEALTH_BAR_OFFSET_Y = -36;
```

Comment:
I'm not sure about this position change.
Player will see only allies health bar (#142) so for the others (enemies) will see only the name.
Without health bars it looks like this
![image](https://user-images.githubusercontent.com/980955/122164009-5d32b880-ce76-11eb-94c2-da4e373f460f.png)

The space between names and the players is quite large

Reply:
This should be resolved in the ticket addressing health bar visibility.